release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1279,6 +1279,29 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
 
 * Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])
 
+* Previously, a webhook for the SR-IOV Network Operator blocked network policies for OpenShift installations on OpenStack.
+Users were not able to create SR-IOV network policies.
+This update fixes the webhook.
+Users can now create SR-IOV network policies for installations on OpenStack.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2016334[*BZ#2016334*])
+
+* Previously, the CRI-O runtime engine passed pod UIDs by using the `K8S_POD_UID` variable.
+But when pods were deleted and recreated at the same time that Multus was setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
```

Comment:
```suggestion
But when pods were deleted and recreated in concurence with Multus setting up networking for the deleted pod's sandbox, this method resulted in additional metadata and unnecessary processing.
```

Reply:
this sentence bothered me last review, and I left it. However, this time, I'd like to suggest an alternative wording to shorten it a little.